content/trial_getting_started/set_trial_manager/other-deployments.md

```diff
@@ -7,36 +7,36 @@ alwaysopen = false
+## {{< param product_name >}} Premium (Premium download / Hosted service trial)
 
 Current Version: 5.1       [Release Notes](https://cloudify.co/release-notes-5-1-2020/)
 
 #### Downloads
 
-Cloudify **Premium Manager** Images:  [RPM](https://repository.cloudifysource.org/cloudify/5.1.0/ga-release/cloudify-manager-install-5.1.0ga.rpm)	|	[Docker](https://repository.cloudifysource.org/cloudify/5.1.0/ga-release/cloudify-manager-aio-docker-5.1.0ga.tar)	|	[QCOW](https://repository.cloudifysource.org/cloudify/5.1.0/ga-release/cloudify-manager-5.1.0ga.qcow2)
+{{< param product_name >}} **Premium Manager** Images:  [RPM](https://repository.cloudifysource.org/cloudify/5.1.0/ga-release/cloudify-manager-install-5.1.0ga.rpm)	|	[Docker](https://repository.cloudifysource.org/cloudify/5.1.0/ga-release/cloudify-manager-aio-docker-5.1.0ga.tar)	|	[QCOW](https://repository.cloudifysource.org/cloudify/5.1.0/ga-release/cloudify-manager-5.1.0ga.qcow2)
 
-Cloudify **Premium CLI**: [RPM](https://repository.cloudifysource.org/cloudify/5.1.0/ga-release/cloudify-cli-5.1.0-ga.el7.x86_64.rpm)	|	[DEB](https://repository.cloudifysource.org/cloudify/5.1.0/ga-release/cloudify-cli_5.1.0-ga_amd64.deb)	|	[EXE](https://repository.cloudifysource.org/cloudify/5.1.0/ga-release/cloudify-windows-cli_5.1.0-ga.exe)
+{{< param product_name >}} **Premium CLI**: [RPM](https://repository.cloudifysource.org/cloudify/5.1.0/ga-release/cloudify-cli-5.1.0-ga.el7.x86_64.rpm)	|	[DEB](https://repository.cloudifysource.org/cloudify/5.1.0/ga-release/cloudify-cli_5.1.0-ga_amd64.deb)	|	[EXE](https://repository.cloudifysource.org/cloudify/5.1.0/ga-release/cloudify-windows-cli_5.1.0-ga.exe)
 
-By downloading Cloudify Premium, you agree to the [End User License Agreement for Premium version](https://cloudify.co/license).
+By downloading {{< param product_name >}} Premium, you agree to the [End User License Agreement for Premium version](https://cloudify.co/license).
 
 ---
 
 
-## Cloudify Community
+## {{< param product_name >}} Community
 
 Current Version: 5.1  (20.03.18)       [Release Notes](https://cloudify.co/release-notes-5-1-2020/)
 
 #### Downloads
 
-Cloudify **Community Manager** Images:  [RPM](https://repository.cloudifysource.org/cloudify/20.03.18/release/cloudify-manager-install-20.03.18-community.x86_64.rpm)	|	[Docker](https://repository.cloudifysource.org/cloudify/20.03.18/release/cloudify-manager-aio-docker-20.03.18.tar)	|	[QCOW](https://repository.cloudifysource.org/cloudify/20.03.18/community-release/cloudify-manager-community-20.03.18.qcow2)
+{{< param product_name >}} **Community Manager** Images:  [RPM](https://repository.cloudifysource.org/cloudify/20.03.18/release/cloudify-manager-install-20.03.18-community.x86_64.rpm)	|	[Docker](https://repository.cloudifysource.org/cloudify/20.03.18/release/cloudify-manager-aio-docker-20.03.18.tar)	|	[QCOW](https://repository.cloudifysource.org/cloudify/20.03.18/community-release/cloudify-manager-community-20.03.18.qcow2)
```

Comment:
Maybe parameter for community version could be used here (`mgr_community_title`).

Reply:
Consider adding more params in a follow-up review.